libs/SoloPlay/BuildFiles/Runewords/Lore.js

```diff
@@ -18,8 +18,8 @@ if (!Check.haveItem("helm", "runeword", "Lore")) {
 	}
 }
 
-var classLoreHelm = [];
-var loreHelm = [
+const classLoreHelm = [];
```

Comment:
this needs to be changed to let, const variables need to be declared and defined in the same statement

Reply:
ty. done